### ISSUE TITLE
🐛 Bug Report: appwrite v0.12.0.200 is unable to check for a newer version

### ISSUE THREAD

**Issue by acalatrava** [appwrite/appwrite#2602]
### 👟 Reproduction steps

Running `docker exec appwrite doctor` it will report `Failed to check for a newer version` everytime. 

### 👍 Expected behavior

Be able to check for a new version

### 👎 Actual Behavior

Also on the logs you can see this:

```
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:03+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:04+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:04+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:04+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:04+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:05+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:05+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:05+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:05+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:07+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:07+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:07+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:07+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:08+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:08+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:08+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:09+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:09+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:09+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:09+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:10+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:10+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:10+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:10+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:11+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:11+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:11+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T05:54:11+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [2022-01-13 06:09:41 *19.1]	NOTICE	ProcessFactory::finish() (ERRNO 1005): session#166 does not exists
appwrite                        | [2022-01-13 06:09:41 *19.1]	NOTICE	ProcessFactory::end() (ERRNO 1005): session#166 is closed
appwrite                        | [Error] Timestamp: 2022-01-13T06:51:19+00:00
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T07:15:45+00:00
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T08:16:13+00:00
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T09:01:13+00:00
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [2022-01-13 09:06:16 *22.4]	NOTICE	ProcessFactory::finish() (ERRNO 1005): session#169 does not exists
appwrite                        | [2022-01-13 09:06:16 *22.4]	NOTICE	ProcessFactory::end() (ERRNO 1005): session#169 is closed
appwrite                        | [Error] Timestamp: 2022-01-13T09:06:17+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | Log pushed with status code: 403
appwrite                        | Log pushed with status code: 403
appwrite                        | [Error] Timestamp: 2022-01-13T09:06:26+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | [Error] Timestamp: 2022-01-13T09:07:17+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | Log pushed with status code: 403
appwrite                        | [Error] Timestamp: 2022-01-13T09:14:50+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | [Error] Timestamp: 2022-01-13T09:15:32+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | Log pushed with status code: 403
appwrite                        | [Error] Timestamp: 2022-01-13T09:15:38+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | Log pushed with status code: 403
appwrite                        | [Error] Timestamp: 2022-01-13T09:15:41+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T09:15:41+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
appwrite                        | [Error] Timestamp: 2022-01-13T09:17:01+00:00
appwrite                        | [Error] Method: GET
appwrite                        | [Error] URL: /console/version
appwrite                        | [Error] Type: Exception
appwrite                        | Log pushed with status code: 403
appwrite                        | [Error] Message: Failed to check for a newer version
appwrite                        | [Error] File: /usr/src/code/app/controllers/web/console.php
appwrite                        | [Error] Line: 425
appwrite                        | [Error] Timestamp: 2022-01-13T09:19:01+00:00
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Not Found
appwrite                        | [Error] File: /usr/src/code/vendor/utopia-php/framework/src/App.php
appwrite                        | [Error] Line: 719
```
 

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

Linux

### 🧱 Your Environment

Latest version v:0.12.0.200

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by eldadfux** [comment]
Seems like we should hit `appwrite.io` to check for the version and not the local instance.

![image](https://user-images.githubusercontent.com/1297371/149319205-45489f14-11b8-4e80-9812-bbb0eb961790.png)

**Comment by eldadfux** [comment]
Fixed. This patch will be released with the next Appwrite version 0.12.2 or 0.13.0

**Comment by TorstenDittmann** [comment]
We just released 0.13 and this should be fixed 👍🏻 

You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).

Thank you a lot for reporting this 🙏🏻
